pkg/kubernetes/config/kubeconfig.go

```diff
@@ -151,12 +151,16 @@ func RenameConfig(sp *specs.Specs, newConfig *clientcmdapi.Config) {
 	newConfig.Clusters[sp.GetClusterName()] = newConfig.Clusters[DefaultClusterName]
 	delete(newConfig.Clusters, DefaultClusterName)
 
+	log.Debug("Renaming user")
```

Comment:
I see we are calling sp.GetClusterName() 8 times.  I believe it is better to create a var for this value.

Reply:
Agreed 👍